readme.md

```diff
@@ -127,7 +127,7 @@ Option accepts `function` with `retry` and `error` arguments. Function must retu
 
 ###### followRedirect
 
-Type: `boolean`  
```

Comment:
@luanmuniz this will remove line breake before `Default`. You can see it in md preview.